src/modules/billing/service/charge-processor/model-mapper.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * This maps the data output from the charge processor
+ * to service models
+ */
+const { Address, Batch, Invoice, InvoiceAccount } = require('../../../../lib/models');
+const { uniqBy } = require('lodash');
+
+/**
+ * Maps address data from CRM to water service Address model
+ * @param {Object} data - address data from CRM
+ * @return {Address}
+ */
+const mapAddress = data => {
+  const address = new Address();
+  address.id = data.addressId;
+  address.addressLine1 = data.address1;
+  address.addressLine2 = data.address2;
+  address.addressLine3 = data.address3;
+  address.addressLine4 = data.address4;
```

Comment:
Is there a danger here that `data.address4` might be undefined and cause a Joi validation error which requires a string or null?

Reply:
The data here comes from the CRM API call so will always be a string or null